TestPlaybooks/playbook-UnEscapeURLs-Test.yml

```diff
@@ -630,7 +631,73 @@ tasks:
     view: |-
       {
         "position": {
-          "x": -380,
+          "x": 3490,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+  "24":
+    id: "24"
+    taskid: 7155da1f-73a7-47bb-8416-00cd05169a4b
+    type: regular
+    task:
+      id: 7155da1f-73a7-47bb-8416-00cd05169a4b
+      version: -1
+      name: UnEscapeURL - backward slash
+      description: |-
+        Extract URLs redirected by security tools like Proofpoint.
+        Changes https://urldefense.proofpoint.com/v2/url?u=https-3A__example.com_something.html -> https://example.com/something.html
+        Also, un-escape URLs that are escaped for safety with formats like hxxps://www[.]demisto[.]com
+      scriptName: UnEscapeURLs
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "25"
+    scriptarguments:
+      input:
+        simple: http:\\example.com
```

Comment:
Add also a test for no slash case: `http:example.com`

Reply:
added the specified testcase